pkg/internal/controller/controller.go

```diff
@@ -220,7 +221,6 @@ func (c *Controller[request]) Start(ctx context.Context) error {
 			return err
 		}
 
-		// Start the SharedIndexInformer factories to begin populating the SharedIndexInformer caches
```

Comment:
You mentioned that the comment is in the wrong place and I ended up removing it entirely, as a source doesn't really necessitate a SharedIndexInformer at this point and I found it to be mostly just confusing

Reply:
Absolutely makes sense!